packages/server/modules/notifications/services/handlers/activityDigest.ts

```diff
@@ -8,7 +8,7 @@ import {
   Activity,
   AllActivityTypes,
   StreamScopeActivity
-} from '@/modules/activitystream/services/types'
+} from '@/modules/activitystream/helpers/types'
```

Comment:
coverage is still quite low overall (-3%), but im guessing you're already working on this

Reply:
yup, now its better...